python/google/__init__.py

```diff
@@ -1 +1 @@
-__import__('pkg_resources').declare_namespace(__name__)
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
I'm not a package expert in Python, but I was running into issues here with package namespace conflicts in the Docker container. Basically, Python was importing the `google` namespace only from the generated code `google.transit`, but wasn't finding `google.protobuf` in `site-packages`. I was able to resolve this in the container run by extending the path instead... but I'm not sure if this will break anything when actually publishing the package. I would love someone with more language domain knowledge here to chime in! (Or advise if I'm just doing something wrong)